xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -135,7 +135,7 @@ class CAddonRecording : public PVR_RECORDING
     strEpisodeName = m_episodeName.c_str();
     iSeriesNumber = recording.m_iSeason;
     iEpisodeNumber = recording.m_iEpisode;
-    iEpisodePartNumber = PVR_RECORDING_INVALID_SERIES_EPISODE; //! @todo
+    iEpisodePartNumber = m_episodePartNumber;
```

Comment:
This needs to read `iEpisodePartNumber = recording.m_episodePartNumber;` or `iEpisodePartNumber = recording.GetEpisodePart();`